scripts/valhalla_build_timezones

```diff
@@ -21,7 +21,7 @@ fi
 rm -rf dist
 rm -f ./timezones-with-oceans.shapefile.zip
 
-url="https://github.com/evansiroky/timezone-boundary-builder/releases/download/2024a/timezones-with-oceans.shapefile.zip"
+url="https://github.com/evansiroky/timezone-boundary-builder/releases/download/2025a/timezones-with-oceans-1970.shapefile.zip"
```

Comment:
see https://github.com/valhalla/valhalla/pull/5162#issuecomment-2746558281

Reply:
how did valhalla previously deal with deprecated timezones in the spatialite db? I assume they were filtered out? If so, should the filter be removed as well?

Reply:
Before we had that other map that I now removed and they generally didn't hurt:) now, whatever is in that db also has to exist in the single map that is left, or we'll our sanity check when initializing the tzdb